docs/reference/search/search-your-data/search-application-api.asciidoc

```diff
@@ -36,11 +36,121 @@ Learn more by reading about <<search-template,search templates>>.
 
 If no template is stored with a search application, a minimal <<search-application-api-default-template, default search template>> will be applied at search time.
 The default template implements a simple search use case.
-You can check your query parameters against the current template using the <<search-application-render-query,Render search application query>> API call.
+
+To create a search application with the default template, issue a <<put-search-application, create or update Search Application>> request without specifying a template:
+
+// NOTE for test setup: The create search application command will return a warning
+// when a template is not specified. This will break docs tests.
+// Therefore the setup specifies the default search template
+// even though we're using the default template in examples.
```

Comment:
```suggestion
// even though we're not using the default template in examples.
```

Did you mean "not using the default template"

Reply:
No, we're not specifying a template in the examples therefore using the default template.